templates/default/supervisor-control.erb

```diff
@@ -17,6 +17,10 @@ SUPERVISOR_PID_WORKERS=`$JAVA_HOME/bin/jps -l |grep "backtype.storm.daemon.worke
 # Check what the caller wants.
 case "$1" in
     start)
+        if [ -r /etc/environment ]; then
+          source /etc/environment
+        fi
+
```

Comment:
Now that I think of it you'll probably also need to append this right after the `fi`:
```bash
export VAULT_TOKEN
```